src/app/Http/Controllers/CrudController.php

```diff
@@ -118,7 +118,9 @@ public function storeCrud(StoreRequest $request = null)
         $this->data['entry'] = $this->crud->entry = $item;
 
         // show a success message
-        \Alert::success(trans('backpack::crud.insert_success'))->flash();
+        if($this->crud->model::find($item->id))
```

Comment:
If this were to be accepted, it would need braces for the then clause.

Reply:
Totally agree with this should be done, thank you @ihsanberahim . I personally haven't EVER managed to break the inserting, but if we do and we reach this conditional, I'd say:
- we also need an ```else``` with "_An error has occured. The entry has NOT been added._" message;
- we'll need to do the same with the updateCrud() method;
- we'll need the ```if () {} else {}``` braces, like @lloy0076 said;

@ihsanberahim would you like to continue working on this PR with the above, so we can merge it faster and count as a contribution for you on Github? If you don't have the time, I totally understand, we'll make a note of it and continue for you when we have the time.

Thanks, cheers!